src/settings.rs

```diff
@@ -17,8 +20,17 @@ pub struct DnsSettings {
 
 impl DnsSettings {
   pub fn load(filename: String) -> Result<Self, Box<dyn Error>> {
-    let contents = fs::read_to_string(shellexpand::full(filename.as_str()).unwrap().to_string())
-      .expect("Aw man, there was an issue while opening the config file :(");
+
+    let filenames= [filename.as_str(), "./dns.config.yaml", "~/.config/simpledns/dns.config.yaml", "/etc/simpledns/dns.config.yaml"];
+    let mut contents = String::new();
+    for filename in filenames {    
+      if !Path::new(filename).exists() { continue }
+      let error_str = "Aw man, there was an issue while opening the config file '{".to_owned() + filename + "}' :(";
+      contents = fs::read_to_string(shellexpand::full(filename).unwrap().to_string())
```

Comment:
Contents shouldnt need to be in the loop here

Reply:
Done in fdc210a3426c85b54f86e0d3948a1001bda38a57